File: Step 1 - Learn the Basics/1.4 Know Basic Maths/03 Check Palindrome/palindrome.java
// https://leetcode.com/problems/palindrome-number/

class Solution {
    public boolean isPalindrome(int x) {
        if(x<0)
            return false;
        long ans = 0;
        long com = x;
        while (x != 0) {
            ans = ans * 10 + x % 10;
            x /= 10;
        }
        return ans == com ? true: false;
    }
}

File: Step 1 - Learn the Basics/1.4 Know Basic Maths/03 Check Palindrome/palindrome.py
# https://leetcode.com/problems/palindrome-number/

class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        com = str(x)
        temp = str(x)[::-1]
        if com == temp:
            return True
        return False